package.json

```diff
@@ -155,13 +157,14 @@
     "request": "^2.81.0",
     "rethink-knex-adapter": "^0.4.18",
     "rollbar": "^2.4.4",
+    "style-loader": "^1.2.1",
     "thinky": "^2.3.3",
     "timezonecomplete": "^5.5.0",
     "twilio": "^3.40.0",
     "url-join": "^4.0.1",
     "uuid": "^3.1.0",
     "wait-for-expect": "^1.1.1",
-    "webpack": "^3.8.1",
+    "webpack": "^4.44.1",
```

Comment:
will need to confirm that this version of webpack works in deploying, but if so, this upgrade would be very welcome!